ffcx/ir/representation.py

```diff
@@ -422,8 +423,37 @@ def _compute_integral_ir(form_data, form_index, prefix, element_numbers, integra
             scheme = md["quadrature_rule"]
             degree = md["quadrature_degree"]
 
-            (points, weights) = create_quadrature_points_and_weights(integral_type, cell, degree,
-                                                                     scheme)
+            if scheme == "custom":
+                points = md["quadrature_points"]
+                weights = md["quadrature_weights"]
```

Comment:
Should there be some checks here, (1) that the points and weights have been provided (and if not, return a useful message), (2) that they are the same length?